src/FSharp.AWS.DynamoDB/FSharp.AWS.DynamoDB.fsproj

```diff
@@ -1,21 +1,19 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
IIRC in some strange cases, you need to explicitly specify the framework if you have a single-item list...